examples/discovery-search-app/cypress/integration/passages/spec.ts

```diff
@@ -13,9 +13,11 @@ describe('Passage Results', () => {
 
     // SearchResults displays ONLY the first passage text of the results that have passages
     cy.findByText(
-      'This result multiple passages, but you should only be able to see the first one.'
+      "This result multiple passages, so we'll display them in the order that they're returned."
```

Comment:
Generally, our linter is set up to prefer single quotes, but if there's a single quote in the string itself, the convention is to use double quotes rather than escaping with `/'`.

From the prettier docs:
> If the number of quotes outweighs the other quote, the quote which is less used will be used to format the string - Example: "I'm double quoted" results in "I'm double quoted" and "This \"example\" is single quoted" results in 'This "example" is single quoted'.

https://prettier.io/docs/en/options.html#quotes